### PR TITLE
Nicolas - SF Automation - LPD-36750 

### PR DESCRIPTION
**Pull request by kyle-miho**
https://liferay.atlassian.net/browse/LPD-36750
Forwarded from: https://github.com/liferay-devtools/liferay-portal/pull/245
cc: @nicolas-sss @drewbrokke @rubenssoares-br @ling-alan-huang

### CODE REVIEW

**Comment by kyle-miho** [comment]
@ling-alan-huang @nicolas-sss @drewbrokke Can you all check if my sf check adjustment is okay when you have the chance? The problem is that the source formatter thinks that regex: from checks have parameters to be compared for ordering if there is an open paren symbol present, but technically regex from checks should never be used for comparison order, so I had it skip the check and thus move to the to: clause. This fixed the problem from me locally. Im not sure if theres anything else I am missing

File: modules/util/source-formatter/src/main/resources/dependencies/replacements.json [hunk 1]
```diff
@@ -116,7 +116,7 @@
 			"Http",
 			"HttpUtil"
 		],
-		"from": "regex:(http|_http|HttpUtil)\\.(?<methodName>addParameter|decodePath|decodeURL|encodeParameters|encodePath|fixPath|getCompleteURL|getDomain|getIpAddress|getParameter|getParameterMap|getPath|getProtocol|getQueryString|getRequestURL|getURI|hasDomain|hasProtocol|isForwarded|isSecure|normalizePath|parameterMapFromString|protocolize|removeDomain|removeParameter|removePathParameters|removeProtocol|sanitizeHeader|setParameter|shortenURL)",
+		"from": "regex:(\\w+|HttpUtil)\\.(?<methodName>addParameter|decodePath|decodeURL|encodeParameters|encodePath|fixPath|getCompleteURL|getDomain|getIpAddress|getParameter|getParameterMap|getPath|getProtocol|getQueryString|getRequestURL|getURI|hasDomain|hasProtocol|isForwarded|isSecure|normalizePath|parameterMapFromString|protocolize|removeDomain|removeParameter|removePathParameters|removeProtocol|sanitizeHeader|setParameter|shortenURL)",
```
Comment (ling-alan-huang) [inline modules/util/source-formatter/src/main/resources/dependencies/replacements.json:119]: @kyle-miho 
`"from": "regex:(\\w+|HttpUtil)`: I think `\\w+` can also match `HttpUtil`, so `HttpUtil` is not needed here, I didn't test it though, please check it again. Thanks.

**Comment by kyle-miho** [comment]
ci:test:sf

**Review by drewbrokke — APPROVED** [review]
LGTM, thanks @kyle-miho

**Review by nicolas-sss — APPROVED** [review]

**Comment by drewbrokke** [comment]
ci:forward

**Comment by drewbrokke** [comment]
ci:forward

**Comment by kyle-miho** [comment]
Was manually forwarded and merged here https://github.com/brianchandotcom/liferay-portal/pull/156323

**Pull request closed without merging**